### ISSUE TITLE
Tiến độ tiến hành chương Recurrent Neural Networks

### ISSUE THREAD

**Issue by duythanhvn** [aivivn/d2l-vn#1441]
Mình hy vọng chương **Recurrent Neural Networks** này (54 phần) chúng ta có thể hoàn thiện trước 15/06/2020.

## Giai đoạn 1 `37 parts`
* [x] 8.0 [Recurrent Neural Networks Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `01 part`
* [x] 8.1. [Sequence Models](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sequence_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `06 part`
* [x] 8.2. [Text Preprocessing](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+text-preprocessing_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `03 part`
* [x] 8.3. [Language Models and the Dataset](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+language-models-and-dataset_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `07 part`
* [x] 8.4. [Recurrent Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+rnn_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `06 part`
* [x] 8.5. [Implementation of Recurrent Neural Networks from Scratch](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+rnn-scratch_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `06 part`
* [x] 8.6. [Concise Implementation of Recurrent Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+rnn-gluon_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `02 part`
* [x] 8.7. [Backpropagation Through Time](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+bptt_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `06 part`

## Giai đoạn 2 `17 parts`
* [x] 8.0 [Recurrent Neural Networks Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `01 part`
* [x] 8.1. [Sequence Models](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sequence_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `03 part`
* [x] 8.2. [Text Preprocessing](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+text-preprocessing_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `01 part`
* [x] 8.3. [Language Models and the Dataset](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+language-models-and-dataset_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `03 part`
* [x] 8.4. [Recurrent Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+rnn_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `03 part`
* [x] 8.5. [Implementation of Recurrent Neural Networks from Scratch](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+rnn-scratch_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `03 part`
* [x] 8.6. [Concise Implementation of Recurrent Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+rnn-gluon_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `01 part`
* [x] 8.7. [Backpropagation Through Time](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+bptt_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+recurrent-neural-networks%22) `02 part`

**Comment by tiepvupsu** [comment]
Chương này nhiều nhỉ, phải làm nhanh mới kịp 15/6.

**Comment by rootonchair** [comment]
Phần dịch của chương này gần xong rồi anh.
